msteams-platform/tabs/how-to/authentication/auth-flow-tab.md

```diff
@@ -21,13 +23,13 @@ For example, the authentication flow for tabs and bots using Node and the [OAuth
 This section uses Azure AD as an example of a third party OAuth provider for enabling authentication in a tab app.
 
 > [!NOTE]
-> Before showing a **Login** button to the user and calling the `microsoftTeams.authentication.authenticate` API in response to selecting the button, you must wait for the SDK initialization to complete. You can pass a callback to the `microsoftTeams.initialize` API that is called when initialization completes.
+> Before showing a **Login** button to the user and calling the `authentication.authenticate` API in response to selecting the button, you must wait for the SDK initialization to complete. You can chain a `.then()` handler or `await` for the `app.initialize()` function to complete.
 
 ![Tab authentication sequence diagram](~/assets/images/authentication/tab_auth_sequence_diagram.png)
```

Comment:
The image is not clear or legible, please provide better image.

Reply:
Done--text size is larger now